xbmc/DynamicDll.cpp

```diff
@@ -25,17 +25,19 @@
 
 using namespace XFILE;
 
-DllDynamic::DllDynamic()
+DllDynamic::DllDynamic():
```

Comment:
Yes I like it also with : on next line. I followed a discussion in the past between @fritsch and @FernetMenta  and kept in mind that they want to have it in the first line. but maybe I mix up things :-) So if I'm not the only one who likes it in line 2. I'll love to change it